Jenkinsfile.release

```diff
@@ -96,15 +96,19 @@ spec:
 
         steps {
           container('azure-cli') {
-            sh "az login --service-principal \
-              -u '${ env.AZURE_VAULT_CLIENT_ID }' \
-              -p '${ env.AZURE_VAULT_CLIENT_SECRET }' \
-              -t '${ env.AZURE_VAULT_TENANT_ID }'"
+            sh '''
```

Comment:
Avoid passing interpolated strings to `sh`. [Discussion](https://github.com/jenkinsci/credentials-binding-plugin/blob/22fcd37c62bd880bcacec1b59065ec222aee67d5/src/main/resources/org/jenkinsci/plugins/credentialsbinding/impl/BindingStep/help.html#L28-L42)

Reply:
Good to know thanks